  INFO root [2019-09-09 19:21:21,592] - 
  INFO root [2019-09-09 19:21:21,592] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 19:21:21,726] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 19:21:21,726] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 19:21:21,726] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 19:21:21,820] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 19:21:21,821] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 19:21:21,824] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56746 )
  INFO root [2019-09-09 19:21:21,825] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56746, SOMAXCONN=128.
  INFO root [2019-09-09 19:21:21,825] - server-configs:
  INFO root [2019-09-09 19:21:21,825] - 	internalTcpAddr : 192.168.56.1:56746
  INFO root [2019-09-09 19:21:21,825] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 19:21:21,825] - 	componentID : 9000
  WARN root [2019-09-09 19:21:22,064] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 19:21:22,277] - Components::findLogger: found logger, addr:192.168.56.1:56744
  INFO root [2019-09-09 19:21:22,287] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 19:21:22,287] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 19:21:22,313] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 8032 ----
  INFO root [2019-09-09 19:21:22,313] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 19:21:22,332] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 19:21:22,332] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 19:21:22,567] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 19:21:22,567] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 19:21:22,840] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 19:21:22,840] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 19:21:23,575] - ---- loginapp is running ----
 DEBUG root [2019-09-09 19:21:23,577] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 19:21:29,860] - Forwarding logs to logger[192.168.56.1:56744/0/0/0]...
 ERROR root [2019-09-09 19:22:51,537] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=59748729773, lastReceivedTime=59743582839, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 19:22:51,539] - The following logs sent to logger failed:
  WARN root [2019-09-09 19:22:51,539] - ==> [2019-09-09 19:22:51 534] TCPPacketReceiver::processPendingEvents(192.168.56.1:56744): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:22:51,539] - ==> [2019-09-09 19:22:51 537] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=59748729773, lastReceivedTime=59743582839, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-09 19:22:51,608] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56745): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:22:51,610] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=59748894266, lastReceivedTime=59744480690, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 19:22:51,614] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 19:22:51,615] - onLoginAppShutDown()
  INFO root [2019-09-09 19:22:51,615] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 19:22:51,615] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 19:22:51,624] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 19:22:51,624] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 19:22:51,925] - ThreadPool::destroy(): waiting for thread(2)[0x2c43d0,0x2c45d8,], try=1
  INFO root [2019-09-09 19:22:53,249] - 
  INFO root [2019-09-09 19:22:53,268] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 19:22:53,269] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 19:22:53,270] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 19:22:53,270] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 19:22:53,315] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 19:22:53,316] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 19:22:53,321] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56807 )
  INFO root [2019-09-09 19:22:53,321] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56807, SOMAXCONN=128.
  INFO root [2019-09-09 19:22:53,321] - server-configs:
  INFO root [2019-09-09 19:22:53,322] - 	internalTcpAddr : 192.168.56.1:56807
  INFO root [2019-09-09 19:22:53,322] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 19:22:53,322] - 	componentID : 9000
  WARN root [2019-09-09 19:22:53,566] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 19:22:53,784] - Components::findLogger: found logger, addr:192.168.56.1:56801
  INFO root [2019-09-09 19:22:53,784] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 19:22:53,784] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 19:22:53,789] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 932 ----
  INFO root [2019-09-09 19:22:53,790] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 19:22:53,790] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 19:22:53,790] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 19:22:54,068] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 19:22:54,068] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 19:22:54,080] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 19:22:54,080] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 19:22:54,510] - ---- loginapp is running ----
 DEBUG root [2019-09-09 19:22:54,511] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 19:23:00,614] - Forwarding logs to logger[192.168.56.1:56801/0/0/0]...
 ERROR root [2019-09-09 19:28:30,477] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=60511346208, lastReceivedTime=60495118800, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 19:28:30,480] - The following logs sent to logger failed:
  WARN root [2019-09-09 19:28:30,480] - ==> [2019-09-09 19:28:30 474] TCPPacketReceiver::processPendingEvents(192.168.56.1:56801): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:28:30,481] - ==> [2019-09-09 19:28:30 478] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=60511346208, lastReceivedTime=60495118800, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-09 19:28:30,573] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56802): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:28:30,574] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=60511563733, lastReceivedTime=60495231172, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 19:28:30,575] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 19:28:30,575] - onLoginAppShutDown()
  INFO root [2019-09-09 19:28:30,576] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 19:28:30,576] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 19:28:30,585] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 19:28:30,585] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 19:28:30,886] - ThreadPool::destroy(): waiting for thread(2)[0x192188,0x1921f0,], try=1
  INFO root [2019-09-09 19:28:32,247] - 
  INFO root [2019-09-09 19:28:32,247] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 19:28:32,248] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 19:28:32,249] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 19:28:32,249] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 19:28:32,266] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 19:28:32,266] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 19:28:32,270] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56887 )
  INFO root [2019-09-09 19:28:32,270] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56887, SOMAXCONN=128.
  INFO root [2019-09-09 19:28:32,270] - server-configs:
  INFO root [2019-09-09 19:28:32,270] - 	internalTcpAddr : 192.168.56.1:56887
  INFO root [2019-09-09 19:28:32,270] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 19:28:32,270] - 	componentID : 9000
  WARN root [2019-09-09 19:28:32,506] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-09 19:28:32,896] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20214, errorno=10048
  INFO root [2019-09-09 19:28:32,900] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 3108 ----
  INFO root [2019-09-09 19:28:32,900] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 19:28:32,900] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 19:28:32,901] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 19:28:33,093] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 19:28:33,094] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 19:28:33,105] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 19:28:33,106] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 19:28:33,516] - ---- loginapp is running ----
 DEBUG root [2019-09-09 19:28:33,516] - Components::process(): Request for the process of identity...
  INFO root [2019-09-09 19:28:40,651] - Components::findComponents: find logger(1)...
  INFO root [2019-09-09 19:28:40,732] - Components::findComponents: found logger, addr:192.168.56.1:56882
  INFO root [2019-09-09 19:28:40,733] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 19:28:40,733] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 19:28:40,734] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-09 19:28:40,736] - Components::findComponents: found dbmgr, addr:192.168.56.1:56883
  INFO root [2019-09-09 19:28:40,737] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 19:28:40,737] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 19:28:40,739] - Components::findComponents: found logger, addr:192.168.56.1:56882
  INFO root [2019-09-09 19:28:40,742] - Components::findComponents: found dbmgr, addr:192.168.56.1:56883
  WARN root [2019-09-09 19:28:41,743] - Forwarding logs to logger[192.168.56.1:56882/0/0/0]...
 ERROR root [2019-09-09 19:30:33,194] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=60787458318, lastReceivedTime=60785198879, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-09 19:30:33,201] - The following logs sent to logger failed:
  WARN root [2019-09-09 19:30:33,201] - ==> [2019-09-09 19:30:33 191] TCPPacketReceiver::processPendingEvents(192.168.56.1:56882): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:30:33,201] - ==> [2019-09-09 19:30:33 194] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=60787458318, lastReceivedTime=60785198879, inactivityExceptionPeriod=134887197)
  WARN root [2019-09-09 19:30:33,288] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56883): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:30:33,291] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=60787676895, lastReceivedTime=60785234527, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-09 19:30:33,293] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 19:30:33,294] - onLoginAppShutDown()
  INFO root [2019-09-09 19:30:33,294] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 19:30:33,294] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 19:30:33,304] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 19:30:33,304] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 19:30:33,605] - ThreadPool::destroy(): waiting for thread(2)[0xab1000,0xab13a8,], try=1
  INFO root [2019-09-09 19:30:35,108] - 
  INFO root [2019-09-09 19:30:35,116] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 19:30:35,117] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 19:30:35,117] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 19:30:35,117] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 19:30:35,138] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 19:30:35,139] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 19:30:35,143] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:56996 )
  INFO root [2019-09-09 19:30:35,143] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:56996, SOMAXCONN=128.
  INFO root [2019-09-09 19:30:35,143] - server-configs:
  INFO root [2019-09-09 19:30:35,143] - 	internalTcpAddr : 192.168.56.1:56996
  INFO root [2019-09-09 19:30:35,143] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 19:30:35,143] - 	componentID : 9000
  WARN root [2019-09-09 19:30:35,440] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 19:30:35,822] - Components::findLogger: found logger, addr:192.168.56.1:57000
  INFO root [2019-09-09 19:30:35,823] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 19:30:35,823] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 19:30:35,831] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 12944 ----
  INFO root [2019-09-09 19:30:35,836] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 19:30:35,837] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 19:30:35,838] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 19:30:36,076] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 19:30:36,077] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 19:30:36,090] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 19:30:36,090] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 19:30:36,521] - ---- loginapp is running ----
 DEBUG root [2019-09-09 19:30:36,522] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 19:30:42,648] - Forwarding logs to logger[192.168.56.1:57000/0/0/0]...
 ERROR root [2019-09-09 19:55:13,982] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=64119234294, lastReceivedTime=64099403332, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-09 19:55:13,988] - The following logs sent to logger failed:
  WARN root [2019-09-09 19:55:13,988] - ==> [2019-09-09 19:55:13 979] TCPPacketReceiver::processPendingEvents(192.168.56.1:57000): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:55:13,988] - ==> [2019-09-09 19:55:13 983] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=64119234294, lastReceivedTime=64099403332, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-09 19:55:14,078] - TCPPacketReceiver::processPendingEvents(192.168.56.1:56998): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 19:55:14,081] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=64119455848, lastReceivedTime=64099081019, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-09 19:55:14,087] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 19:55:14,088] - onLoginAppShutDown()
  INFO root [2019-09-09 19:55:14,088] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 19:55:14,088] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 19:55:14,097] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 19:55:14,097] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 19:55:14,398] - ThreadPool::destroy(): waiting for thread(2)[0x2642ec8,0x2643a28,], try=1
  INFO root [2019-09-09 20:21:34,962] - 
  INFO root [2019-09-09 20:21:34,963] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 20:21:34,964] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 20:21:34,964] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 20:21:34,964] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 20:21:34,981] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 20:21:34,982] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 20:21:34,985] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:57606 )
  INFO root [2019-09-09 20:21:34,986] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:57606, SOMAXCONN=128.
  INFO root [2019-09-09 20:21:34,986] - server-configs:
  INFO root [2019-09-09 20:21:34,986] - 	internalTcpAddr : 192.168.56.1:57606
  INFO root [2019-09-09 20:21:34,986] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 20:21:34,986] - 	componentID : 9000
  WARN root [2019-09-09 20:21:35,225] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 20:21:36,730] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 9796 ----
  INFO root [2019-09-09 20:21:36,731] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 20:21:36,731] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 20:21:36,731] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 20:21:36,993] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 20:21:36,993] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 20:21:37,004] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 20:21:37,004] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 20:21:37,438] - ---- loginapp is running ----
 DEBUG root [2019-09-09 20:21:37,439] - Components::process(): Request for the process of identity...
  INFO root [2019-09-09 20:21:44,564] - Components::findComponents: find logger(1)...
  INFO root [2019-09-09 20:21:44,664] - Components::findComponents: found logger, addr:192.168.56.1:57609
  INFO root [2019-09-09 20:21:44,664] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 20:21:44,664] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 20:21:44,666] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-09 20:21:44,670] - Components::findComponents: found logger, addr:192.168.56.1:57609
  INFO root [2019-09-09 20:21:44,673] - Components::findComponents: found dbmgr, addr:192.168.56.1:57611
  INFO root [2019-09-09 20:21:44,673] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 20:21:44,673] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 20:21:44,775] - Components::findComponents: found dbmgr, addr:192.168.56.1:57611
  WARN root [2019-09-09 20:21:45,776] - Forwarding logs to logger[192.168.56.1:57609/0/0/0]...
 ERROR root [2019-09-09 20:26:01,590] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=68276354756, lastReceivedTime=68216674731, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-09 20:26:01,592] - The following logs sent to logger failed:
  WARN root [2019-09-09 20:26:01,592] - ==> [2019-09-09 20:26:01 588] TCPPacketReceiver::processPendingEvents(192.168.56.1:57609): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:26:01,592] - ==> [2019-09-09 20:26:01 591] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=68276354756, lastReceivedTime=68216674731, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-09 20:26:01,690] - TCPPacketReceiver::processPendingEvents(192.168.56.1:57611): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:26:01,690] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=68276579462, lastReceivedTime=68216756522, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-09 20:26:01,712] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 20:26:01,715] - onLoginAppShutDown()
  INFO root [2019-09-09 20:26:01,715] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 20:26:01,715] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 20:26:01,728] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 20:26:01,728] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 20:26:02,029] - ThreadPool::destroy(): waiting for thread(2)[0x228d8d8,0x228d390,], try=1
  INFO root [2019-09-09 20:26:03,438] - 
  INFO root [2019-09-09 20:26:03,439] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 20:26:03,440] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 20:26:03,440] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 20:26:03,440] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 20:26:03,457] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 20:26:03,458] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 20:26:03,463] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:57687 )
  INFO root [2019-09-09 20:26:03,463] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:57687, SOMAXCONN=128.
  INFO root [2019-09-09 20:26:03,464] - server-configs:
  INFO root [2019-09-09 20:26:03,464] - 	internalTcpAddr : 192.168.56.1:57687
  INFO root [2019-09-09 20:26:03,464] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 20:26:03,464] - 	componentID : 9000
  WARN root [2019-09-09 20:26:03,677] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 20:26:03,985] - Components::findLogger: found logger, addr:192.168.56.1:57681
  INFO root [2019-09-09 20:26:03,986] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 20:26:03,986] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 20:26:03,991] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 13856 ----
  INFO root [2019-09-09 20:26:03,992] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 20:26:03,992] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 20:26:03,992] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 20:26:04,186] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 20:26:04,186] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 20:26:04,199] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 20:26:04,199] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 20:26:04,577] - ---- loginapp is running ----
 DEBUG root [2019-09-09 20:26:04,577] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 20:26:10,686] - Forwarding logs to logger[192.168.56.1:57681/0/0/0]...
 ERROR root [2019-09-09 20:33:23,463] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=69270567911, lastReceivedTime=69225262110, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 20:33:23,466] - The following logs sent to logger failed:
  WARN root [2019-09-09 20:33:23,466] - ==> [2019-09-09 20:33:23 462] TCPPacketReceiver::processPendingEvents(192.168.56.1:57681): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:33:23,466] - ==> [2019-09-09 20:33:23 464] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=69270567911, lastReceivedTime=69225262110, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-09 20:33:23,558] - TCPPacketReceiver::processPendingEvents(192.168.56.1:57686): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:33:23,563] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=69270793311, lastReceivedTime=69225456346, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 20:33:23,582] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 20:33:23,584] - onLoginAppShutDown()
  INFO root [2019-09-09 20:33:23,584] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 20:33:23,584] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 20:33:23,598] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 20:33:23,598] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 20:33:23,899] - ThreadPool::destroy(): waiting for thread(2)[0x878758,0x8785b8,], try=1
  INFO root [2019-09-09 20:34:10,015] - 
  INFO root [2019-09-09 20:34:10,015] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 20:34:10,016] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 20:34:10,017] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 20:34:10,017] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 20:34:10,032] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 20:34:10,032] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 20:34:10,036] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:57779 )
  INFO root [2019-09-09 20:34:10,036] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:57779, SOMAXCONN=128.
  INFO root [2019-09-09 20:34:10,037] - server-configs:
  INFO root [2019-09-09 20:34:10,037] - 	internalTcpAddr : 192.168.56.1:57779
  INFO root [2019-09-09 20:34:10,037] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 20:34:10,037] - 	componentID : 9000
  WARN root [2019-09-09 20:34:10,258] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 20:34:11,770] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 13320 ----
  INFO root [2019-09-09 20:34:11,771] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 20:34:11,772] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 20:34:11,772] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 20:34:12,015] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 20:34:12,015] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 20:34:12,032] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 20:34:12,033] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 20:34:12,505] - ---- loginapp is running ----
 DEBUG root [2019-09-09 20:34:12,506] - Components::process(): Request for the process of identity...
  INFO root [2019-09-09 20:34:19,618] - Components::findComponents: find logger(1)...
  INFO root [2019-09-09 20:34:19,662] - Components::findComponents: found logger, addr:192.168.56.1:57777
  INFO root [2019-09-09 20:34:19,663] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 20:34:19,663] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 20:34:19,664] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-09 20:34:19,766] - Components::findComponents: found dbmgr, addr:192.168.56.1:57782
  INFO root [2019-09-09 20:34:19,767] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 20:34:19,767] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 20:34:19,767] - Components::findComponents: found dbmgr, addr:192.168.56.1:57782
  WARN root [2019-09-09 20:34:20,767] - Forwarding logs to logger[192.168.56.1:57777/0/0/0]...
 ERROR root [2019-09-09 20:35:51,035] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=69602604485, lastReceivedTime=69577648581, inactivityExceptionPeriod=134887980)
  INFO root [2019-09-09 20:35:51,042] - The following logs sent to logger failed:
  WARN root [2019-09-09 20:35:51,042] - ==> [2019-09-09 20:35:51 031] TCPPacketReceiver::processPendingEvents(192.168.56.1:57777): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:35:51,043] - ==> [2019-09-09 20:35:51 036] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=69602604485, lastReceivedTime=69577648581, inactivityExceptionPeriod=134887980)
  WARN root [2019-09-09 20:35:51,154] - TCPPacketReceiver::processPendingEvents(192.168.56.1:57782): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:35:51,155] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=69602875469, lastReceivedTime=69578119760, inactivityExceptionPeriod=134887980)
  INFO root [2019-09-09 20:35:51,167] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 20:35:51,170] - onLoginAppShutDown()
  INFO root [2019-09-09 20:35:51,171] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 20:35:51,172] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 20:35:51,195] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 20:35:51,195] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 20:35:51,496] - ThreadPool::destroy(): waiting for thread(2)[0x15d148,0x15d1b0,], try=1
  INFO root [2019-09-09 20:44:38,192] - 
  INFO root [2019-09-09 20:44:38,192] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 20:44:38,193] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 20:44:38,194] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 20:44:38,194] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 20:44:38,211] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 20:44:38,211] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 20:44:38,215] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:57892 )
  INFO root [2019-09-09 20:44:38,216] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:57892, SOMAXCONN=128.
  INFO root [2019-09-09 20:44:38,216] - server-configs:
  INFO root [2019-09-09 20:44:38,216] - 	internalTcpAddr : 192.168.56.1:57892
  INFO root [2019-09-09 20:44:38,216] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 20:44:38,216] - 	componentID : 9000
  WARN root [2019-09-09 20:44:38,425] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 20:44:38,668] - Components::findLogger: found logger, addr:192.168.56.1:57889
  INFO root [2019-09-09 20:44:38,668] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 20:44:38,669] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 20:44:38,673] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 12752 ----
  INFO root [2019-09-09 20:44:38,674] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 20:44:38,674] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 20:44:38,674] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 20:44:38,874] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 20:44:38,874] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 20:44:38,885] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 20:44:38,885] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 20:44:39,271] - ---- loginapp is running ----
 DEBUG root [2019-09-09 20:44:39,271] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 20:44:45,383] - Forwarding logs to logger[192.168.56.1:57889/0/0/0]...
 ERROR root [2019-09-09 20:57:37,047] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=72541132362, lastReceivedTime=72476097469, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 20:57:37,052] - The following logs sent to logger failed:
  WARN root [2019-09-09 20:57:37,052] - ==> [2019-09-09 20:57:37 040] TCPPacketReceiver::processPendingEvents(192.168.56.1:57889): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:57:37,052] - ==> [2019-09-09 20:57:37 047] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=72541132362, lastReceivedTime=72476097469, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-09 20:57:37,171] - TCPPacketReceiver::processPendingEvents(192.168.56.1:57887): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 20:57:37,179] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=72541429173, lastReceivedTime=72476060022, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 20:57:37,200] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 20:57:37,201] - onLoginAppShutDown()
  INFO root [2019-09-09 20:57:37,201] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 20:57:37,201] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 20:57:37,214] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 20:57:37,214] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 20:57:37,515] - ThreadPool::destroy(): waiting for thread(2)[0x2695a8,0x269470,], try=1
  INFO root [2019-09-09 20:57:39,205] - 
  INFO root [2019-09-09 20:57:39,211] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 20:57:39,212] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 20:57:39,213] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 20:57:39,213] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 20:57:39,258] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 20:57:39,258] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 20:57:39,263] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58044 )
  INFO root [2019-09-09 20:57:39,264] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58044, SOMAXCONN=128.
  INFO root [2019-09-09 20:57:39,264] - server-configs:
  INFO root [2019-09-09 20:57:39,264] - 	internalTcpAddr : 192.168.56.1:58044
  INFO root [2019-09-09 20:57:39,264] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 20:57:39,265] - 	componentID : 9000
  WARN root [2019-09-09 20:57:39,521] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 20:57:39,776] - Components::findLogger: found logger, addr:192.168.56.1:58040
  INFO root [2019-09-09 20:57:39,776] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 20:57:39,776] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 20:57:39,784] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 12916 ----
  INFO root [2019-09-09 20:57:39,785] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 20:57:39,786] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 20:57:39,786] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 20:57:40,152] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 20:57:40,153] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 20:57:40,176] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 20:57:40,176] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 20:57:40,636] - ---- loginapp is running ----
 DEBUG root [2019-09-09 20:57:40,637] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 20:57:46,781] - Forwarding logs to logger[192.168.56.1:58040/0/0/0]...
 ERROR root [2019-09-09 21:00:41,005] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=72955038693, lastReceivedTime=72950870593, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-09 21:00:41,007] - The following logs sent to logger failed:
  WARN root [2019-09-09 21:00:41,008] - ==> [2019-09-09 21:00:41 003] TCPPacketReceiver::processPendingEvents(192.168.56.1:58040): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 21:00:41,008] - ==> [2019-09-09 21:00:41 005] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=72955038693, lastReceivedTime=72950870593, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-09 21:00:41,098] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58037): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 21:00:41,100] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=72955252304, lastReceivedTime=72950854331, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-09 21:00:41,104] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 21:00:41,105] - onLoginAppShutDown()
  INFO root [2019-09-09 21:00:41,105] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 21:00:41,105] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 21:00:41,114] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 21:00:41,114] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 21:00:41,415] - ThreadPool::destroy(): waiting for thread(2)[0xc54580,0xc543e0,], try=1
  INFO root [2019-09-09 21:00:42,780] - 
  INFO root [2019-09-09 21:00:42,783] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 21:00:42,784] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 21:00:42,785] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 21:00:42,785] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 21:00:42,813] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 21:00:42,813] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 21:00:42,817] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58086 )
  INFO root [2019-09-09 21:00:42,817] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58086, SOMAXCONN=128.
  INFO root [2019-09-09 21:00:42,818] - server-configs:
  INFO root [2019-09-09 21:00:42,818] - 	internalTcpAddr : 192.168.56.1:58086
  INFO root [2019-09-09 21:00:42,818] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 21:00:42,818] - 	componentID : 9000
  WARN root [2019-09-09 21:00:43,041] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 21:00:43,286] - Components::findLogger: found logger, addr:192.168.56.1:58083
  INFO root [2019-09-09 21:00:43,286] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 21:00:43,286] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 21:00:43,292] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 704 ----
  INFO root [2019-09-09 21:00:43,292] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 21:00:43,292] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 21:00:43,292] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 21:00:43,496] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 21:00:43,496] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 21:00:43,507] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 21:00:43,508] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 21:00:43,892] - ---- loginapp is running ----
 DEBUG root [2019-09-09 21:00:43,892] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 21:00:49,999] - Forwarding logs to logger[192.168.56.1:58083/0/0/0]...
 ERROR root [2019-09-09 21:04:50,122] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=73515552906, lastReceivedTime=73499095863, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 21:04:50,126] - The following logs sent to logger failed:
  WARN root [2019-09-09 21:04:50,126] - ==> [2019-09-09 21:04:50 116] TCPPacketReceiver::processPendingEvents(192.168.56.1:58083): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 21:04:50,127] - ==> [2019-09-09 21:04:50 123] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=73515552906, lastReceivedTime=73499095863, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-09 21:04:50,226] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58084): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 21:04:50,231] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=73515798351, lastReceivedTime=73499080270, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-09 21:04:50,235] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 21:04:50,239] - onLoginAppShutDown()
  INFO root [2019-09-09 21:04:50,239] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 21:04:50,239] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 21:04:50,252] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 21:04:50,252] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 21:04:50,553] - ThreadPool::destroy(): waiting for thread(2)[0x22a4808,0x22a4600,], try=1
  INFO root [2019-09-09 21:04:52,068] - 
  INFO root [2019-09-09 21:04:52,086] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 21:04:52,087] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 21:04:52,088] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 21:04:52,088] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 21:04:52,132] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 21:04:52,133] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 21:04:52,137] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58173 )
  INFO root [2019-09-09 21:04:52,137] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58173, SOMAXCONN=128.
  INFO root [2019-09-09 21:04:52,138] - server-configs:
  INFO root [2019-09-09 21:04:52,138] - 	internalTcpAddr : 192.168.56.1:58173
  INFO root [2019-09-09 21:04:52,138] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 21:04:52,138] - 	componentID : 9000
  WARN root [2019-09-09 21:04:52,362] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 21:04:52,661] - Components::findLogger: found logger, addr:192.168.56.1:58171
  INFO root [2019-09-09 21:04:52,661] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 21:04:52,661] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 21:04:52,666] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 5160 ----
  INFO root [2019-09-09 21:04:52,667] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 21:04:52,667] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 21:04:52,667] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 21:04:52,896] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 21:04:52,896] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 21:04:52,910] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 21:04:52,911] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 21:04:53,345] - ---- loginapp is running ----
 DEBUG root [2019-09-09 21:04:53,345] - Components::process(): Request for the process of identity...
  WARN root [2019-09-09 21:04:59,479] - Forwarding logs to logger[192.168.56.1:58171/0/0/0]...
 ERROR root [2019-09-09 21:07:09,105] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=73828265517, lastReceivedTime=73790015196, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-09 21:07:09,109] - The following logs sent to logger failed:
  WARN root [2019-09-09 21:07:09,109] - ==> [2019-09-09 21:07:09 104] TCPPacketReceiver::processPendingEvents(192.168.56.1:58171): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 21:07:09,109] - ==> [2019-09-09 21:07:09 106] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=73828265517, lastReceivedTime=73790015196, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-09 21:07:09,199] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58168): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-09 21:07:09,200] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=73828478717, lastReceivedTime=73790008830, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-09 21:07:09,206] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-09 21:07:09,206] - onLoginAppShutDown()
  INFO root [2019-09-09 21:07:09,206] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-09 21:07:09,206] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-09 21:07:09,222] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-09 21:07:09,223] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-09 21:07:09,524] - ThreadPool::destroy(): waiting for thread(2)[0x1a60b8,0x1a6598,], try=1
  INFO root [2019-09-09 21:07:10,940] - 
  INFO root [2019-09-09 21:07:10,956] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-09 21:07:10,957] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-09 21:07:10,958] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-09 21:07:10,958] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-09 21:07:10,991] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-09-09 21:07:10,991] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-09-09 21:07:10,995] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58254 )
  INFO root [2019-09-09 21:07:10,996] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58254, SOMAXCONN=128.
  INFO root [2019-09-09 21:07:10,996] - server-configs:
  INFO root [2019-09-09 21:07:10,996] - 	internalTcpAddr : 192.168.56.1:58254
  INFO root [2019-09-09 21:07:10,996] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-09-09 21:07:10,996] - 	componentID : 9000
  WARN root [2019-09-09 21:07:11,211] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-09 21:07:11,502] - Components::findLogger: found logger, addr:192.168.56.1:58253
  INFO root [2019-09-09 21:07:11,502] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-09 21:07:11,503] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-09 21:07:11,508] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 2082. PID: 7524 ----
  INFO root [2019-09-09 21:07:11,508] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-09 21:07:11,508] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-09 21:07:11,509] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-09 21:07:11,729] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-09 21:07:11,730] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-09 21:07:11,741] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-09 21:07:11,741] - TelnetServer server is running on port 31000
  INFO root [2019-09-09 21:07:12,133] - ---- loginapp is running ----
 DEBUG root [2019-09-09 21:07:12,133] - Components::process(): Request for the process of identity...
